docs/guides/cloud/integrations/slack.mdx

```diff
@@ -231,3 +146,22 @@ will be able to install the Cypress integration for Slack again in the future.
 2. Within the Slack Integration, click **Configure**.
 3. Click **Uninstall Slack Integration** to uninstall the Cypress integration
    for Slack.
+
+## Security and compliance
+
+Review our [Privacy Policy](https://on.cypress.io/privacy-policy) for information about how the Cypress app for Slack collects and
+manages your information.
+
+### Authorizations
+
+When authorized, Cypress can:
+
+- View information about a user’s identity
```

Comment:
Can we be more specific here? This sounds really broad. Like, you can see the user's first name/last name? Or their entire profile?

Reply:
This is what Slack shares when reviewing the app
![Screenshot 2024-09-25 at 10 12 56 AM](https://github.com/user-attachments/assets/96fe891b-ac54-43bd-a988-d68b9a1eb9c6)